setupKAppNavLib.sh

```diff
@@ -28,8 +29,34 @@ handleParams() {
 
     # set defaults
     doBuild=false
-    doPush=false
     
+    allBuildArguments=("$@")
+    for ((index=0; index < ${#allBuildArguments[@]}; index++)); do
+        if [ ${allBuildArguments[index]} = '-r' ]; then
+            reposArg=${allBuildArguments[index+1]}
+            if [ "$reposArg" = "" ]; then
+                echo "Missing required argument for -r"
+                exit 1 # exit script with error
+            fi
+            # parse the r option values
+            IFS=',' read -r -a repos <<< "$reposArg"
+        fi
+    done
+
+    # construct projectList to pass in to build and pushimages, and imageOption variable to pass in to install
```

Comment:
This comment isn't correct any more.  pushImages takes an image list, not project list.